Makefile

```diff
@@ -7,14 +7,14 @@ all: install-tools compile-schema validate-example
 compile-schema:
 	@if ! npm ls ajv-cli; then npm install; fi
 	@for f in $(SCHEMA_FILES); do \
-	    npx --no ajv-cli compile --spec=draft2020 -s $$f \
+	    npx --no ajv-cli compile --spec=draft2020 -s ./schema/$$f -r "./schema/!($$f)" \
```

Comment:
This change makes all the schemas in the `/schema/` dir (excluding the schema being compiled) available for reference by the schema being compile.